### PR TITLE
Remove deprecated compliance and security related frontend API

### PR DESCRIPTION
**Pull request by alyiwang**
rename frontend API to /compliance

### CODE REVIEW

File: wherehows-frontend/conf/routes [hunk 1]
```diff
@@ -140,17 +140,9 @@ POST          /api/v1/datasets/:id/watch
 
 DELETE        /api/v1/datasets/:id/watch/:watchId                               controllers.api.v1.Dataset.unwatchDataset(id:Int, watchId:Int)
 
-GET           /api/v1/datasets/:id/compliance                                   controllers.api.v1.Dataset.getDatasetCompliance(id:Int)
+GET           /api/v1/datasets/:id/privacy                                      controllers.api.v1.Dataset.getDatasetCompliance(id:Int)
```
Comment (mars-lan) [inline wherehows-frontend/conf/routes]: Can we just call it compliance instead of privacy?

Comment (alyiwang) [inline wherehows-frontend/conf/routes]: Yes, I was going to talk to @theseyi about this. Currently we are using this privacy API, it makes more sense to rename to compliance, but the UI needs to rename the API calls also.

Comment (mars-lan) [inline wherehows-frontend/conf/routes]: Let's do 1 rename and do it right :)

Comment (theseyi) [inline wherehows-frontend/conf/routes]: We can rename it

**Pull request merged**